config/features.php

```diff
@@ -17,5 +17,6 @@
     'voter_reg_modal' => env('DS_ENABLE_VOTER_REG_MODAL', false),
     'dynamic_explore_campaigns' => env('DS_ENABLE_DYNAMIC_CAMPAIGNS_PAGE', false),
     'volunteer_credits' => env('DS_ENABLE_VOLUNTEER_CREDITS', false),
-    'cause_preferences' => env('DS_ENABLE_CAUSE_INTERESTS_PAGE', false),
+    'voter_reg_drive_total' => env('DS_ENABLE_VOTER_REG_DRIVE_TOTAL', false),
+    'cause_preferences' => env('DS_ENABLE_CAUSE_INTERESTS_PAGE', false)
```

Comment:
@Dinnall Looks like the linter just wants a trailing comma on this line!
```suggestion
    'cause_preferences' => env('DS_ENABLE_CAUSE_INTERESTS_PAGE', false),
```

Reply:
That's very odd, thanks. Going forth should I place trailing commas at the end of all arrays?

Reply:
Yeah, we use [StyleCI](https://styleci.io/) to ensure a consistent implementation of some conventional coding styling in our PHP code (and heck yeah it can be annoying sometimes lol but it's pretty worth it for a neat, readable codebase!) You can see the presets we use and some rules we've set [here](https://github.com/DoSomething/phoenix-next/blob/0623ddc2a5c1f4982d30d7ecec1010b23e38877f/.styleci.yml#L1) in the config.